env.d.ts

```diff
@@ -840,7 +854,7 @@ type Highsec = Fullsec & PlayerHighsec & {
 		}) => (
 			Omit<UpgradeCore, keyof F | `rarity`> & F & {
 				[x: string]: null | boolean | number | string
-				rarity: "`0noob`" | "`1kiddie`" | "`2h4x0r`" | "`3h4rdc0r3`" | "`4|_|b3|2`" | "`531337`"
+				rarity: UpgradeRarity
```

Comment:
this is supposed to be `UpgradeRarityString` because you only receive this result type when you pass `is_script: false` which causes `sys.upgrades` return return colour coded strings

Reply:
Apologise, missed this one!